posts/heatmaps-clustering/index.qmd

```diff
@@ -39,13 +47,14 @@ from scipy.cluster import hierarchy
 
 ## Data
 
-Let us quickly get the presidential addresses from nltk, and then compute the Jaccard distance on the 5-shingles.
+Let us quickly get the presidential addresses from nltk, and then compute the Jaccard distance on the 5-shingles[^1].
+
+[^1]: Note that the data itself is not the focus here. We just need a distance matrix in order to show how it can be reordered. We have chosen to work with a matrix representing the distances between texts in a given corpus, and to consider Jaccard distance on shingles. _Shingles_ are substrings of a certain length that are created by passing a moving window along the text. This choice of distance between texts is common when working with web data, and can deal with mispellings and sequential nature of text (to some extent). See e.g. [@schutze2008introduction, Section 19.6].
 
 First we download and import the data.
 
 ```{python}
 #| eval: false
-import nltk
 nltk.download("inaugural")
 ```
 
```

Comment:
Can't comments further down but:
1) straight forward -> straightforward
2) there's some commented out code, like the use of pandas `.loc`. Do you want to keep that?
3) you mention "optimal ordering" -- maybe worth a few more words on what you mean?
4) after you plot by data and by distance you could maybe spare a few words for a conclusion/description of the results?